preupg/xmlgen/compose.py

```diff
@@ -77,8 +77,44 @@ def get_compose_dir_name(self):
 class ComposeXML(object):
 
     @staticmethod
```

Comment:
Make this method a non-static so you don't need to pass the _module_set_dir_ and _generate_from_ini_ to the method.